reference/api/pubsub_api.md

```diff
@@ -109,7 +110,7 @@ SUCCESS | message is processed successfully
 RETRY | message to be retried by Dapr
 DROP | warning is logged and message is dropped
```

Comment:
Others | error, message to be retried.

Reply:
@artursouza  no .... message will not be retried on any other status ....

Reply:
Will fix that PR in a minute ...